TRENZ.Lib.RazorMail.Core/Extensions/RazorMailServiceCollectionExtensions.cs

```diff
@@ -7,13 +9,17 @@ namespace TRENZ.Lib.RazorMail.Extensions;
 
 public static class RazorMailServiceCollectionExtensions
 {
-    public static IServiceCollection AddRazorEmailRenderer(this IServiceCollection services)
+    [Obsolete("Use AddRazorMailRenderer instead.")]
+    public static IServiceCollection AddRazorEmailRenderer(this IServiceCollection services) =>
+        services.AddRazorMailRenderer();
+
+    public static IServiceCollection AddRazorMailRenderer(this IServiceCollection services)
     {
         services.AddMvcCore()
-                .AddRazorViewEngine();
+            .AddRazorViewEngine();
 
-        services.AddTransient<IMailRenderer, MailRenderer>();
+        services.AddTransient<IMailRenderer, RazorMailRenderer>();
```

Comment:
OK, but then shouldn't it also be `IRazorMailRenderer`?

Reply:
See other comment